configureWorkspace/configure.ts

```diff
@@ -399,8 +398,8 @@ interface PackageJson {
     artifactName: string
 }
 
-async function getPackageJson(folder: vscode.WorkspaceFolder): Promise<vscode.Uri[]> {
-    return vscode.workspace.findFiles(new vscode.RelativePattern(folder, 'package.json'), null, 1, null);
+async function getPackageJson(folderPath: string): Promise<vscode.Uri[]> {
+    return vscode.workspace.findFiles(new vscode.RelativePattern(folderPath, 'package.json'), null, 1, null);
```

Comment:
Using Uri instead of string is a more interesting suggestion. But personally not a fan of using it for paths that you know are file paths, because Uri carries the idea that it might be a URL, so then do you have to assert that it's a file path before calling fsPath?  Because right now the code makes that assumption whenever it calls fsPath, so it might as well be explicit from the start that it's not a URL.

Node.js already handles both '/' and '\' interchangeably and as long as you are careful to use Node.js functions (e.g. path.join, path.basename) instead of using string manipulation, everything works fine.